1.0

aliliolek opened the pull request:
- [DEMO LINK](https://aliliolek.github.io/layout_miami/)

stanislav-topikha reviewed (changes requested):
MOST of the points from checklist didn't implemented
please check it again and fix it
https://github.com/mate-academy/layout_miami/blob/master/checklist.md

liliia-livandovskaya reviewed (changes requested):
1. When I hover over the menu items i don't see the line under the item. Add it as it shows on the mockup
<img width="541" alt="Знімок екрана 2022-08-26 о 11 11 56" src="https://user-images.githubusercontent.com/104769168/186856436-31c78b7d-3d4c-4db8-a172-49d37d06bcff.png">
2. When i hover over the phone icon there is no phone number to see 
<img width="298" alt="Знімок екрана 2022-08-26 о 11 12 25" src="https://user-images.githubusercontent.com/104769168/186856812-0d3c572a-6958-41d2-bf5b-08185cffc9cf.png">

3. On submitting the form there should not be any error. Read the checklist to fix it  
<img width="717" alt="Знімок екрана 2022-08-26 о 11 14 06" src="https://user-images.githubusercontent.com/104769168/186856967-1b7ef255-b7ef-44da-bb16-1180777a7f87.png">

4. Make the hover effect smaller in this blick because it hovers the next one
<img width="1035" alt="Знімок екрана 2022-08-26 о 11 13 02" src="https://user-images.githubusercontent.com/104769168/186857121-9669ddc7-47ac-4e44-bbf9-e935f20d2acc.png">
5.  the contact us form looks different for the Tablet, please check and fix it 
<img width="418" alt="Знімок екрана 2022-08-26 о 11 16 37" src="https://user-images.githubusercontent.com/104769168/186857360-d186e2c3-9d1b-42f6-a673-135ca31479a4.png">

aliliolek commented:
> 1. When I hover over the menu items i don't see the line under the item. Add it as it shows on the mockup
> 
> <img alt="Знімок екрана 2022-08-26 о 11 11 56" width="541" src="https://user-images.githubusercontent.com/104769168/186856436-31c78b7d-3d4c-4db8-a172-49d37d06bcff.png">
> 
> 2. When i hover over the phone icon there is no phone number to see
> <img alt="Знімок екрана 2022-08-26 о 11 12 25" width="298" src="https://user-images.githubusercontent.com/104769168/186856812-0d3c572a-6958-41d2-bf5b-08185cffc9cf.png">
> 
> 3. On submitting the form there should not be any error. Read the checklist to fix it
> 
> <img alt="Знімок екрана 2022-08-26 о 11 14 06" width="717" src="https://user-images.githubusercontent.com/104769168/186856967-1b7ef255-b7ef-44da-bb16-1180777a7f87.png">
> 
> 4. Make the hover effect smaller in this blick because it hovers the next one
> 
> <img alt="Знімок екрана 2022-08-26 о 11 13 02" width="1035" src="https://user-images.githubusercontent.com/104769168/186857121-9669ddc7-47ac-4e44-bbf9-e935f20d2acc.png">
> 
> 5. the contact us form looks different for the Tablet, please check and fix it
> <img alt="Знімок екрана 2022-08-26 о 11 16 37" width="418" src="https://user-images.githubusercontent.com/104769168/186857360-d186e2c3-9d1b-42f6-a673-135ca31479a4.png">

1 --- I did it now
2 --- I think it works now and worked before 
3 --- I cant see any errors when I tap on "submit"
4 --- I did it now
5 --- (???)

SanyaBratashchuk reviewed (changes requested):
1.Nothing happens when I hover on phone icon or click on it 
![image](https://user-images.githubusercontent.com/97547824/187507509-053e9509-ef5c-4a45-809a-b0e89d46d71d.png)
2.404 after form submit
![image](https://user-images.githubusercontent.com/97547824/187507891-9c8427d9-d911-4071-8a7b-41c24a970198.png)
3.E-mail should be a link to 
![image](https://user-images.githubusercontent.com/97547824/187508014-fc23bcf6-3796-4015-990c-a15682945109.png)

aliliolek commented:
> 1.Nothing happens when I hover on phone icon or click on it ![image](https://user-images.githubusercontent.com/97547824/187507509-053e9509-ef5c-4a45-809a-b0e89d46d71d.png) 2.404 after form submit ![image](https://user-images.githubusercontent.com/97547824/187507891-9c8427d9-d911-4071-8a7b-41c24a970198.png) 3.E-mail should be a link to ![image](https://user-images.githubusercontent.com/97547824/187508014-fc23bcf6-3796-4015-990c-a15682945109.png)

![image](https://user-images.githubusercontent.com/51528776/187770260-5df9ad9a-f312-4458-b5e2-ac928b6e3ed8.png)
1. I think I did it, but on github nothing works, on localhost all's ok.

2. fixed
![image](https://user-images.githubusercontent.com/51528776/187770805-814622fb-c230-4f06-9f90-43f00a532c29.png)


3. fixed
![image](https://user-images.githubusercontent.com/51528776/187770571-7be6b171-df2c-48d8-b6c4-7633309c4b54.png)

SanyaBratashchuk reviewed (changes requested):
Placeholder for message is not visible
![image](https://user-images.githubusercontent.com/97547824/188266962-7a535253-e28d-4489-b29f-6d309367aa1e.png)
When I click on logo it shouldn't open a landing in new tab

aliliolek commented:
> Placeholder for message is not visible ![image](https://user-images.githubusercontent.com/97547824/188266962-7a535253-e28d-4489-b29f-6d309367aa1e.png) When I click on logo it shouldn't open a landing in new tab

![image](https://user-images.githubusercontent.com/51528776/188274228-e3f7f43f-f914-4c6f-9b44-f70770a2c84e.png)
yep, sorry, it was about this problem

liliia-livandovskaya reviewed (changes requested):
Good Job! 
1. There should not be that element on a pic 
<img width="1009" alt="Знімок екрана 2022-09-06 о 11 48 10" src="https://user-images.githubusercontent.com/104769168/188592363-219eef15-5e42-4729-849b-92c756d5534d.png">
2. This should open the contact us page or work as a phone number 
<img width="586" alt="Знімок екрана 2022-09-06 о 11 53 06" src="https://user-images.githubusercontent.com/104769168/188592643-f0eb71ca-1ba3-49f5-9940-88c2ae1e8385.png">
